### PR TITLE
[PFTO] Flatten nested outputs like torch.onnx

### PR DESCRIPTION
**Pull request by take-cheeze**
Ref: https://github.com/pfnet/pytorch-pfn-extras/pull/563
`prim::TupleConstruct` would be flattened by https://sourcegraph.com/github.com/pytorch/pytorch@77d29bcee200f04bece4a86283acfb8e1ec830ad/-/blob/torch/csrc/jit/passes/onnx/peephole.cpp?L765

### CODE REVIEW

**Review by xuzijian629 — APPROVED** [review]
LGTM!!

(confirmation)
The number of input/output values in exported ONNX is not equal to the number of model inputs/outputs. Rather, it equals the number of flattened inputs/outputs.

This PR supports nested outputs. It sets value_info using flattened outputs internally while returns original outputs in export function.

File: tests/pytorch_pfn_extras_tests/onnx_tests/utils.py [hunk 1]
```diff
@@ -51,6 +52,8 @@ def run_model_test(
                     output_names=output_names,
                     **kwargs,
                 )
+                torch_f.flush()
```
Comment (take-cheeze) [inline tests/pytorch_pfn_extras_tests/onnx_tests/utils.py:55]: Windows doens't allow multiple access to file so it needs to be closed instead
```suggestion
                torch_f.close()
```

**Comment by take-cheeze** [comment]
https://ci.preferred.jp/pytorch-pfn-extras.torch110-win/114308/ for windows test check

**Comment by take-cheeze** [comment]
https://ci.preferred.jp/pytorch-pfn-extras.torch110-win/114357/ again

**Comment by take-cheeze** [comment]
https://ci.preferred.jp/pytorch-pfn-extras.torch110-win/114422/

**Pull request merged**